### ISSUE TITLE
[feature request] : when syncing is enabled, allow users to see browser data on android app & android app data via chrome extension

### ISSUE THREAD

**Issue by arooni** [habitlab/habitlab#476]
I consider poor Internet habits to be habits regardless of device.  Would be useful as a user to have a dashboard overview on my phone and on chrome extension or if space/dev time doesn't allow, just the chrome extension that would allow me to see how I'm doing across devices (all browsers + devices).  Not unlike how rescuetime does it.

Thanks!

**Comment by arooni** [comment]
Hi drew; very cool.  

As far as I can tell though; I must have created a goal within the chrome extension to see the mobile app usage (say for facebook for example).

So it seems as though I still cannot see app usage for services I don't have a goal for [say for my twitter app on android which is not the official twitter client].  Would be nice to have this ability.

As well as visualizing all my data on my phone.

Keep up the great work folks!  I've already cut my phone usage by an hour on average thanks to your aggressive interventions.

**Comment by DrewGregory** [comment]
Yes: these are all very real things that we are currently working on. Stay tuned!

Thank you for being such a patient user!
